packages/wonder-blocks-i18n/docs.md

```diff
@@ -0,0 +1,7 @@
+Progress spinners to be used when indicating that some piece of content is loading. They come in three sizes (default, small, and xsmall) and also in light and dark varities. By default `CircularSpinner` should be used for most loading indicators.
+
+```js
+import {CircularSpinner} from "@khanacademy/wonder-blocks-progress-spinner";
+
+<CircularSpinner />
+```
```

Comment:
This is copy/pasta.  I'm going to change this to some placeholder text and will tackle docs in a separate PR (and probably a separate JIRA task).